.github/workflows/presubmit.yml

```diff
@@ -72,26 +71,23 @@ jobs:
         run: |
           cd room && ./gradlew :buildOnServer && cd ../work && ./gradlew :buildOnServer
         id: gradlew_bos
-      - name: Set webhook outputs from ./gradlew
-        if: always()
-        run: |
-          if [ "${{ steps.gradlew_bos.outcome }}" == "success" ] || [ "${{ steps.gradlew_bos.outcome }}" == "skipped" ]
-          then
-            echo "##[set-output name=success;]true"
-          else
-            echo "##[set-output name=success;]false"
-          fi
-        id: gradlew_bos_outputs
       - name: Upload Results
         if: always()
         uses: actions/upload-artifact@v2
         with:
           name: artifacts
           path: ~/dist
-      - name: Completed WebHook
-        if: always()
+      - name: Successful WebHook
+        if: ${{ success() }}
```

Comment:
Is the ${{ }} necessary here?

Reply:
i.e., why isn't it needed for `always()`

Reply:
I just copied the example at https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#job-status-check-functions. 

I think this is something I can probably get rid of.

Reply:
Done.